jenkins/deploy.sh

```diff
@@ -30,22 +30,23 @@ if [ "$DATABRICKS" == true ]; then
     cd deploy
     tar -zxvf ../spark-rapids-built.tgz
     cd spark-rapids
+    echo "chagned to spark-rapids"
 fi
 
-ART_ID=`mvn exec:exec -q -pl $DIST_PL -Dexec.executable=echo -Dexec.args='${project.artifactId}'`
-ART_VER=`mvn exec:exec -q -pl $DIST_PL -Dexec.executable=echo -Dexec.args='${project.version}'`
+echo "Getting maven project artifactid"
+ART_ID=rapids-4-spark_2.12
```

Comment:
I had to hardcode these because the way we build on a databricks host and copy the tgz of it back. mvn complains about no version when you try to run it on the copied back files.